ctwrap/simulation.py

```diff
@@ -471,12 +543,19 @@ def run_parallel(self,
         """
         Run variation using multiprocessing.
 
+        The :meth:`run_parallel` method runs all the variations in the input
+        file in parallel using python multiprocessing. A minimal example is:
```

Comment:
used single back-ticks around multiprocessing and see whether this creates a link. Adding `:any:` in front of it may be necessary.